cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -574,10 +581,27 @@ def compose_trajectory(self, component_name, parameter_name):
 			traj_msg.points.append(point_msg)
 		return (traj_msg, 0)
 
-	def calculate_point_time(self, component_name, start_pos, end_pos, default_vel):
+	def calculate_point_time(elf, component_name, start_pos, end_pos, default_vel, default_acc):
```

Comment:
type: `elf` -> `self`

Reply:
`calculate_point_time` is only used internally (not in external sss API), thus I removed the default again because `vel` doesn't have a default either